proto/internalapi/central/compliance_operator.proto

```diff
@@ -265,3 +265,15 @@ message ComplianceOperatorSuiteV2 {
   reserved 3;
   ComplianceOperatorStatus status = 4;
 }
+
+message ComplianceOperatorRemediationV2 {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string name = 2; // @gotags: search:"Compliance Remediation Name,hidden"
+  // TODO: Name or ID?
+  string compliance_check_result_name = 3; // @gotags: search:"Compliance Check Result Name,hidden"
+  bool apply = 4; // @gotags: search:"Compliance Remediation Applied,hidden"
```

Comment:
```suggestion
  string id = 1;
  string name = 2;
  // TODO: Name or ID?
  string compliance_check_result_name = 3;
  bool apply = 4;
```

not a storage proto, so those tags are not needed.

Reply:
done